browser.py

```diff
@@ -16,7 +16,7 @@ async def spawn_driver(playwright):
 
     Return the created driver.
     """
-    driver = await playwright.webkit.launch(headless=False, handle_sigint=False)
+    driver = await playwright.chromium.launch(headless=False, handle_sigint=False, args=['--use-fake-device-for-media-stream', '--auto-accept-camera-and-microphone-capture'])
```

Comment:
Consider breaking this up into shorter lines:
```suggestion
    driver = await playwright.chromium.launch(
        headless=False,
        handle_sigint=False, 
        args=['--use-fake-device-for-media-stream',
              '--auto-accept-camera-and-microphone-capture'])
```
I wonder how awful it would be to install `black` and run the codebase through that. but that's for another PR in the future, not this one.